src/Tribe/Date_Utils.php

```diff
@@ -572,6 +572,78 @@ public static function is_all_day( $all_day_value ) {
 			);
 		}
 
+		/**
+		 * Determine if "now" is between two dates.
+		 *
+		 * @since TBD
+		 *
+		 * @param string|DateTime|int $start_date A `strtotime` parsable string, a DateTime object or a timestamp.
+		 * @param string|DateTime|int $end_date   A `strtotime` parsable string, a DateTime object or a timestamp.
+		 * @param string|DateTime|int $now        A `strtotime` parsable string, a DateTime object or a timestamp. Defaults to 'now'.
+		 *
+		 * @return boolean
```

Comment:
Add missing doc-block.
I'm not suggesting one here as you've got a way with words I lack.